src/main/java/core/basesyntax/machine/Machine.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.machine;
+
+public abstract class Machine {
+    private String strength = "💪"; //just for fun
+    private String relaxing = "🦥"; //just for fun x2
+

```

Comment:
It's criminal to forbid developers from having fun😉  but don't forget to remove such code before pushing it. Your future team leader may have a bad sense of humor :harold: